UnitySetup/UnitySetup.psm1

```diff
@@ -962,6 +966,7 @@ function Start-UnityEditor {
         if ( $TestResults ) { $sharedArgs += '-testResults', $TestResults }
         if ( $RunTests ) { $sharedArgs += '-runTests' }
         if ( $ForceFree) { $sharedArgs += '-force-free' }
+        if ( $AdditionalArguments) { $sharedArgs += $AdditionalArguments }
```

Comment:
We should have bumped the minor version to 5.1 on the last change, would be nice to get that in now. We won't need a new bump when we move from develop to master as that'll all go in accumulatively. That's in the UnitySetup.psd1 file.